website/_layouts/default.html

```diff
@@ -6,7 +6,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
-    <meta name="description" content="Make Your Company Data Driven. Connect to any data source, easily visualize and share your data.">
+    <meta name="description" content="Make Your Company Data Driven. Connect to any data source, easily query and visualize in moments and share your insights with your team via dashboards and integrations. Query data from multiple sources with our unique Query Results data source. Redash plays with many databases and services including Amazong Redshift, BigQuery, PotsgreSQL, MySQL, TreasureData, MS SQL Server, MongoDB, ElasticSearch, Hive, Google Spreadsheets, Impala, InfluxDB, Presto, ScyllaDB, Cassandra, Amazon DynamoDB, Python, Vertica, Graphite, Stitch. ">
```

Comment:
It needs to be shorted anyway, how about something like: "Use Redash to connect to any data source (Redshift, BigQuery, MySQL, PostgreSQL, MongoDB and many more supported), query, visualize and share your data to make your company data driven."?